indico/modules/events/sessions/__init__.py

```diff
@@ -59,18 +59,6 @@ def _merge_users(target, source, **kwargs):
     SessionPrincipal.merge_users(target, source, 'session')
 
 
-@signals.users.registered.connect
```

Comment:
https://github.com/nop33/indico/commit/fe6934abee4c7557f50208b98a1aaedce4d1fab5#diff-8ec19b9072a58cb01d12d8fef9d1a14dR308

This fix in the same commit should stay (initially I also planned to suggest deleting the commit before I noticed that fix)